go.mod

```diff
@@ -12,136 +12,144 @@ require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mysqlreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/nginxreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/podmanreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/postgresqlreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/rabbitmqreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/riakreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/saphanareceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkhecreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlserverreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/syslogreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/tcplogreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/udplogreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowseventlogreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsperfcountersreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver v0.61.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
-	go.opentelemetry.io/collector v0.60.1-0.20220916163348-84621e483dfb
-	go.opentelemetry.io/collector/pdata v0.60.1-0.20220916163348-84621e483dfb
+	go.opentelemetry.io/collector v0.61.0
+	go.opentelemetry.io/collector/pdata v0.61.0
 	go.uber.org/multierr v1.8.0
 	go.uber.org/zap v1.23.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/aerospike/aerospike-client-go/v6 v6.4.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl v0.61.0 // indirect
+	github.com/prometheus/prometheus v0.38.0 // indirect
+	github.com/signalfx/signalfx-agent/pkg/apm v0.0.0-20201202163743-65b4fa925fc8 // indirect
+)
```

Comment:
I'm curious why this separate block is here?

Reply:
auto generated from `make tidy`. I'll collapse it I prefer just two blocks: direct and indirect